src/styles/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    margin-right: 4px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
```

Comment:
The selector '&__star:nth-child(-n + 4)' is not following the BEM methodology. Instead of applying styles directly to the nth-child, consider using a modifier class to handle the active state of stars.